src/client/localplayer.cpp

```diff
@@ -432,23 +432,51 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	const ContentFeatures &f = nodemgr->get(map->getNode(m_standing_node));
 	const ContentFeatures &f1 = nodemgr->get(map->getNode(m_standing_node + v3s16(0, 1, 0)));
 
+	// We can jump from a bouncy node we collided with this clientstep,
+	// even if we are not "touching" it at the end of clientstep.
+	bool bouncy_can_jump = false;
```

Comment:
Unnecessary variable. You can use `actual_bouncy` (which I suggest to rename to "standing_node_bouncy") value `0` to indicate lack of a bouncy effect.